app/views/shared/views/ems_common/_form.html.haml

```diff
@@ -28,7 +28,7 @@
         - else
           = h(@edit[:ems_types][@edit[:new][:emstype]])
     = render(:partial => "form_fields", :locals => {:url => url})
-    - if @edit[:new][:emstype] && @edit[:new][:emstype] == 'rhevm'
+    - if @edit[:new][:emstype] && %w(rhevm openstack_infra).include?(@edit[:new][:emstype])
```

Comment:
@chessbyte right, though I think that will require bigger refactoring. Also I think this form will be replaced by the angular in the future, right? 

I would prefer this small fix, so the backport is small.